src/static/css/_calls-to-action.css

```
@@ -0,0 +1,125 @@
+.calls-to-action--insight {
```

Set the variables for the default here too:

```suggestion
.calls-to-action,
.calls-to-action--insight
```